mainstyle.css

```diff
@@ -0,0 +1,235 @@
+* {
+  width: 100%;
+  height: 100%;
```

Comment:
usually instead of width and height of 100% in the * selector you only use padding/margin/box-sizing